gator.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17-alpine AS builder
```

Comment:
alpine image doesn't use glibc which can lead to unexpected behavior. can we use `1.18-bullseye`?